### PR TITLE
#353 - Auth tests joined for endpoints requiring read permissions

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #353 
Auth tests joined for endpoints requiring read permissions

### CODE REVIEW

File: src/test/java/com/artipie/docker/http/BaseEntityGetTest.java [hunk 1]
```diff
@@ -57,18 +53,14 @@
...
-        this.slice = new DockerSlice(
-            new AstoDocker(new InMemoryStorage()),
-            new Permissions.Single(TestAuthentication.ALICE.name(), "read"),
-            new TestAuthentication()
-        );
+        this.slice = new DockerSlice(new AstoDocker(new InMemoryStorage()));
     }
 
     @Test
     void shouldRespondOkToVersionCheck() {
```
Comment (genryxy) [inline src/test/java/com/artipie/docker/http/BaseEntityGetTest.java:60]: @olegmoz I'd suggest to remove `SuppressWarnings("PMD.AvoidDuplicateLiterals")` annotation from this class and others

**Review by genryxy — CHANGES_REQUESTED** [review]
@olegmoz thanks, please see one comment

**Comment by olegmoz** [comment]
@genryxy thanks, removed unneeded suppression

**Review by genryxy — APPROVED** [review]
@olegmoz thanks, approve

**Pull request merged**